source/content/drupal-cms.md

```diff
@@ -0,0 +1,135 @@
+
+## Troubleshooting common issues with Drupal CMS
+
+In addition to Recipes, Drupal CMS elevates other new technical constructs within Drupal can be challenging to troubleshoot or raise the importance of workflow clarity.
+
+### Timeouts and Errors during Drupal CMS Installation
+
+Selecting a large number of recipes in during installation seems to correlate with an increased likelihood of timeouts and errors.
+
+If you encounter a timeout or error during installation you can likely resolve it by clearing caches:
+
+  ![Clear Cache Button](../images/clear-cache-button.png)
+
+If clearing caches does not resolve the error, you may need to wipe your database and install again. Wiping can be done via the [dashboard](/site-dashboard) or [Terminus](/terminus/commands/env-wipe).
+
+[See this GitHub issue for more discussion of timeouts and errors during Drupal CMS installation.](https://github.com/pantheon-upstreams/drupal-cms-composer-managed/issues/1)
+
+### Project Browser and file system writablility
+
+Project Browser is a new feature that allows modules and recipes to be installed or applied directly from Drupal's browser-based administrative interface.
+Adding modules and recipes to a site using assumes some version-controlled files and directories are writable by the web server and it also presumes that there is only one web server.
+
+On Pantheon sites, Test and Live environments are locked down for security purposes, and therefore not writable.
+Also, on some plans, Test and Live environments have multiple containers that can serve web requests.
+This means that Project Browser can only be used on Pantheon in the Dev and Multidev environments when those environments are in SFTP mode.
+
+After using Project Browser to install modules, you must [commit and push the changes to your codebase in order to deploy them to Test and Live](/drupal-configuration-management).
+
+### Project Browser and Pantheon-provided Composer scripts
+
+If you intend to use Project Browser first remove extra the composer scripts that Pantheon provides upon site creation. These scripts hook into the Composer lifecycle and can interfere with the Project Browser's ability to install modules and recipes.
+
+These scripts are most useful for teams that then create their own upstreams or distributions.
+They also provide some helper functionality like keeping the PHP versions declared in `composer.json` in sync with those declared in `pantheon.yml`.
+
+To remove these scripts:
+
+1. Delete the `upstream-configuration` entry from the `repositories` section of `composer.json`:
+
+```json
+        {
+            "type": "path",
+            "url": "upstream-configuration"
+        }
+```
+
+2. Delete the `"upstream-configuration` entry from the `require` section of `composer.json`:
+
+
+```json
+        "pantheon-upstreams/upstream-configuration": "dev-main",
+```
+
+3. Delete the `autoload`, `scripts`, and `scripts-descriptions` from `composer.json`:
```

Comment:
```suggestion
3. Delete the `autoload`, `scripts`, and `scripts-descriptions` that reference `upstream-configuration` or `DrupalComposerManaged` from `composer.json`:
```

The user might have their own `autoload` or `scripts`/`scripts-descriptions` in their `composer.json` file and should not just remove those sections wholesale.

Reply:
good point